### PR TITLE
Prepare for 2.0 release

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

**Comment by garyb** [comment]
@paf31 any comments here?

File: src/Data/Maybe/First.purs [hunk 1]
```diff
@@ -53,7 +40,7 @@ instance applicativeFirst :: Applicative First where
   pure = First <<< pure
 
 instance bindFirst :: Bind First where
-  bind (First x) f = First (bind x (runFirst <<< f))
+  bind (First x) f = First (x >>= \y -> case f y of First ma -> ma)
```
Comment (paf31) [inline src/Data/Maybe/First.purs]: I think this should be derivable.

File: src/Data/Maybe/Last.purs [hunk 2]
```diff
@@ -53,7 +40,7 @@ instance applicativeLast :: Applicative Last where
   pure = Last <<< pure
 
 instance bindLast :: Bind Last where
-  bind (Last x) f = Last (bind x (runLast <<< f))
+  bind (Last x) f = Last (x >>= \y -> case f y of Last ma -> ma)
```
Comment (paf31) [inline src/Data/Maybe/Last.purs]: Same

**Comment by paf31** [comment]
Sorry didn't see this one. Nothing blocking, just noticed a couple of possibly-derivable instances. Thanks!

**Comment by garyb** [comment]
Nice, thanks for the reminder - they're almost entirely derivable :smile:

**Comment by paf31** [comment]
They should be marginally faster too, which is good.

**Pull request merged**